### ISSUE TITLE
Logo Proposal

### ISSUE THREAD

**Issue by debpedrano** [drichard/mindmaps#77]
Good day, is there any chance I can make a logo for your project ? it would be interesting and also I can enhance my skills in designing. Thank you !

**Comment by drichard** [comment]
Absolutely, I love the idea! What do you have in mind? The current logo is like 7 years old and never received much love.

Something simple but nice would be fitting I think.

**Comment by debpedrano** [comment]
@drichard 

How about this one ? 
![image](https://user-images.githubusercontent.com/40510200/44043293-18d07a54-9f55-11e8-9316-093da7c0182a.png)

**Comment by drichard** [comment]
Hey, I like the color scheme but the logo seems a bit too complex? It doesn't work well when scaled down to 16px.

Also a nitpick: currently, mindmaps is spelled as all lower case. I like to keep it that way for simplicity.

**Comment by debpedrano** [comment]
@drichard  , ill try some modification of this one. thank you for your response anyways.

**Comment by debpedrano** [comment]
Hello, can I update the issue? please response. Thank you :)
